### DISCUSSION TITLE
Hi, where is the newest beta build to test?

### DISCUSSION THREAD

**Discussion by AshleyPikachu** [PintaProject/Pinta#944] [Q&A]
Hi, I found a 2020 build from the pinta site. Can you pass me a link to the newest one please.

**Comment by cameronwhite** [comment]
https://github.com/PintaProject/Pinta/releases/tag/dev-preview-builds
(although it says 2020, the downloads are periodically updated)

You can also download from specific commits via https://github.com/PintaProject/Pinta/actions

**Reply by AshleyPikachu** [reply]
Thanks for letting me know :)

**Reply by AshleyPikachu** [reply]
Hi again, I went to try it, I was wondering do you have a RPM or flatpak for Rocky 9 for it. That way I don't have to compile it every time it updates. Thanks!

**Reply by cameronwhite** [reply]
There isn't anything set up for that currently, but there was some discussion at https://github.com/flathub/com.github.PintaProject.Pinta/issues/42 about setting up periodic beta releases on flatpak

**Reply by AshleyPikachu** [reply]
Awesome, if that happens it would make testing a lot easier.
